CorsixTH/Lua/app.lua

```diff
@@ -1839,16 +1839,32 @@ function App:checkForUpdates()
   end
 
   local update_table = loadstring_envcall(update_body, "@updatechecker") {}
+  update_table.revision = update_table.revision or 0
```

Comment:
Nice we have built in some wider considerations for how we treat revision. I assume if we go ahead on this path we're going to end up at `0.70.0` or `0.68.0`.